samples/VSSample.Tests/HttpSyncStartTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Moq;
+using FluentAssertions;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json;
+
+namespace VSSample.Tests
+{
+    public class HttpSyncStartTests
+    {
+        private const string FuncitonName = "SampleFunction";
```

Comment:
Typo. `FuncitonName` should be `FunctionName`

Reply:
Thank you for catching that!